src/main/java/net/apnic/rdap/autnum/filters/AutnumRouteFilter.java

```diff
@@ -0,0 +1,50 @@
+package net.apnic.rdap.autnum.filters;
+
+import net.apnic.rdap.authority.RDAPAuthority;
+import net.apnic.rdap.autnum.AsnRange;
+import net.apnic.rdap.directory.Directory;
+import net.apnic.rdap.error.MalformedRequestException;
+import net.apnic.rdap.filter.filters.RDAPPathRouteFilter;
+import net.apnic.rdap.filter.RDAPRequestPath;
+import net.apnic.rdap.filter.RDAPRequestType;
+import net.apnic.rdap.resource.ResourceNotFoundException;
+
+/**
+ *
+ */
+public class AutnumRouteFilter
```

Comment:
I think creating multiple zuul filters that share global state is adding a bit of unnecessary complexity.

Just one zuul filter (i.e. RedirectFilter) that takes the request, takes the resource locators, and returns a redirect is plenty.